zenodb.go

```diff
@@ -270,46 +300,63 @@ func NewDB(opts *DBOpts) (*DB, error) {
 func (db *DB) FlushAll() {
 	db.tablesMutex.Lock()
 	for name, table := range db.tables {
-		log.Debugf("Force flushing table: %v", name)
+		db.log.Debugf("Force flushing table: %v", name)
 		table.forceFlush()
 	}
 	db.tablesMutex.Unlock()
-	log.Debug("Done force flushing tables")
+	db.log.Debug("Done force flushing tables")
+}
+
+// Go starts a goroutine with a task. The task should look for the stop channel to close,
```

Comment:
You'll see this function used in a lot of places. This gives us a standard pattern for stopping goroutines when we close the database.